satpy/etc/composites/visir.yaml

```diff
@@ -647,3 +647,9 @@ composites:
       - wavelength: 3.9
       - wavelength: 6.2
       - wavelength: 1.6
+
+  land_surface_temperature:
+    compositor: !!python/name:satpy.composites.SingleBandCompositor
+    prerequisites:
+      - name: 'lst'
+    standard_name: land_surface_temperature
```

Comment:
I'm tempted to say this shouldn't be in `visir` yet. I'm also tempted to say this compositor should not exist...could you add the standard_name in the reader YAML and/or reader python code? Then this composite doesn't conflict with other readers who already have a product called "land_surface_temperature", but your enhancement based on `standard_name` would still be found.